optuna/study.py

```diff
@@ -109,6 +109,17 @@ def trials(self):
 
         return self._storage.get_all_trials(self.study_id)
 
+    @property
+    def unsafe_trials(self):
```

Comment:
IMO, it's better to add `Study.get_all_trials(self, deepcopy: bool)` method instead of `unsafe_trials`.
Because
- the word `unsafe` is a bit of ambiguous, and
- `Study.trials` property can be regarded as a short form of `Study.get_all_trials(deepcopy=True)`, and
- the method name `get_all_trials` is consistent with the existing storage method (i.e., `BaseStorage.get_all_trials`).

Reply:
Now, I think `get_trials` is a better name than `get_all_trials` (e.g., the former name allows us to add `trial_state` option that filters target trials in the future).